website/content/v1.7/talos-guides/install/cloud-platforms/akamai.md

```diff
@@ -55,21 +57,27 @@ Run the following commands to create three control plane nodes:
 export NODEBALANCER_ID=$(linode-cli nodebalancers list --label talos --format id --text --no-headers)
 export NODEBALANCER_CONFIG_ID=$(linode-cli nodebalancers configs-list ${NODEBALANCER_ID} --format id --text --no-headers)
 export REGION=us-ord
+export LINODE_TYPE=g6-standard-4
+export ROOT_PW=$(pwgen 16)
+
+echo "Setting root password to: $ROOT_PW"
 
 for id in $(seq 3); do
   linode_label="talos-control-plane-${id}"
+
   # create linode
+
   linode-cli linodes create  \
     --no-defaults \
-    --root_pass securepass123! \
-    --type g6-standard-4 \
+    --root_pass ${ROOT_PW} \
```

Comment:
Talos doesn't have a concept of the root password, so I guess this gets ignored/dropped?

Reply:
if so, we might need a note about that

Reply:
Yes, it is a linode API requirement. I can add a note that this is required but not used

Reply:
I've updated the docs with a note calling this out